tests/Agent/IntegrationTests/SharedApplications/Common/MultiFunctionApplicationHelpers/NetStandardLibraries/RabbitMQ.cs

```diff
@@ -34,7 +35,7 @@ class RabbitMQ
         // A SortedDictionary is used to verify that the instrumentation interacts with the message
         // headers through the IDictionary interface.
         // See https://github.com/newrelic/newrelic-dotnet-agent/issues/639 for context
-        private static SortedDictionary<string,object> userHeaders = new SortedDictionary<string, object>() { { "aNumber", 123 }, { "aString", "foo" } };
+        private static IDictionary<string,object> userHeaders = new ReadOnlyDictionary<string, object>(new SortedDictionary<string, object>() { { "aNumber", 123 }, { "aString", "foo" } });
```

Comment:
@tehbio: Are we already testing the other code paths in unit/integration tests? It seems like there is a case where it resolves to null, or a mutable collection. It would be good to test those code paths as well.